Documentation/artifact-format-v3.md

```diff
@@ -200,51 +196,50 @@ Format: JSON
 }
 ```
 
-The `updates` list is a list of all the updates contained within the
-artifact. The intention of having multiple updates is to allow proxy based
-updates to deploy to several different hosts at the same time. However, for
-updates downloaded to single devices, there will usually be only one.
+The `payloads` list is a list of all the Artifact payloads contained within the
+Artifact. The intention of having multiple payloads is to allow multiple updates
+to several distinct components to be contained in one Artifact. For example,
+there may be an update to a file, and a package install contained in the same
+Artifact. For full rootfs updates, there will usually be only one payload.
 
-`type` is the type of update contained within the image. At the moment there is
-only `rootfs-image`, but there may be others in the future, like `docker-image`
-or something package based.
+`type` is the type of payload contained within the image. At the moment there is
```

Comment:
I see what you mean, but I can't think of a good alternative either (the attribute's name is just so right!). I don't think your suggestion improves it much, because you end up referring to "type" as well, after the comma.